lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -67,6 +67,14 @@ function _signup_resource_definition() {
             'source' => ['param' => 'page'],
             'default value' => 1,
           ],
+          [
+            'name' => 'competition',
+            'description' => 'Only competition signups, defaults to 0.',
+            'optional' => TRUE,
+            'type' => 'int',
+            'source' => ['param' => 'competition'],
+            'default value' => 0,
```

Comment:
`FALSE`... just indicated the changes based on my comment regarding using booleans instead :smile: